spec/spec_helper.rb

```diff
@@ -23,4 +24,16 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  config.before(:suite) do
+    # mysql requires truncation
+    DatabaseCleaner.strategy = ENV["DB"].to_s.include?("mysql") ? :truncation : :transaction
+    DatabaseCleaner.clean_with(:truncation)
```

Comment:
Is this correct?  We specify the strategy of `transaction` on line 30 for pg but then `clean_with` is passed `truncation`?

Reply:
yea. so I think they have 2 concepts.
What you should do at the beginning of everything (the `suite`), and what you should do after each transaction.

I did this based upon the bottom of their [how to use](https://github.com/DatabaseCleaner/database_cleaner#how-to-use) section

Reply:
> I did this based upon the bottom of their [how to use](https://github.com/DatabaseCleaner/database_cleaner#how-to-use) section

Thanks, I get it now. Use this strategy going forward but first truncate the db.

Should we swap the order like they have in that section?

![image](https://user-images.githubusercontent.com/19339/235479343-ceea6312-8191-4cb1-b032-f08be152b478.png)